src/Api/Models/Response/SubscriptionResponseModel.cs

```diff
@@ -42,6 +42,7 @@ public SubscriptionResponseModel(User user, UserLicense license = null)
     public BillingSubscription Subscription { get; set; }
     public UserLicense License { get; set; }
     public DateTime? Expiration { get; set; }
+    [Obsolete("In-app purchase payment method is not allowed.")]
     public bool UsingInAppPurchase { get; set; }
```

Comment:
❓ Can I get some info on where this might be used, if anywhere? May not need the obsolete attribute.

Reply:
@withinfocus The only place it's used, as far as I can tell, is in the `user-subscription.component` in `clients` [here](https://github.com/bitwarden/clients/blob/28de9439beb87133c8683434df952a0c0be94100/apps/web/src/app/billing/individual/user-subscription.component.ts#L72), [here](https://github.com/bitwarden/clients/blob/28de9439beb87133c8683434df952a0c0be94100/apps/web/src/app/billing/individual/user-subscription.component.ts#L109) and [here](https://github.com/bitwarden/clients/blob/28de9439beb87133c8683434df952a0c0be94100/apps/web/src/app/billing/individual/user-subscription.component.ts#L172). It just opens certain dialogs restricting the user from managing their subscription a certain way if they're using in-app purchases.

Reply:
Aha, I see now. I killed those off in https://github.com/bitwarden/clients/pull/7433/commits/18768bbf840132ba7f71aa2f0a31eea1c536f4b9. I can probably remove this now that it's gone from there.